samples/mcuboot_config/mcuboot_config.template.h

```diff
@@ -73,6 +73,10 @@
 
 /* Uncomment to use ARM's mbedTLS cryptographic primitives */
 /* #define MCUBOOT_USE_MBED_TLS */
+/* Uncomment to use the MBEDTLS library of version 3.0.0 when
+ * MCUBOOT_USE_MBED_TLS is defined.
+ */
+/* #define MBED_TLS_VERSION_3_0_0 */
```

Comment:
I would suggest not to define a new macro, please use the macros from the `include/mbedtls/build_info.h` header (used to be in the `include/mbedtls/version.h` header before v3.0). You can use the `MBEDTLS_VERSION_NUMBER` for this purpose. See an example here: `boot/bootutil/src/image_rsa.c:103` .

Reply:
Done